
# GITHUB ISSUES REPORT FOR microsoft/pylance-release


Generated on 2024-02-09 using: stale=30, all=False


Only showing items that are new to report in past 1 day(s)


## FOR ISSUES THAT ARE MARKED AS BUGS:


### Issues in pylance-release that have comments from 3rd party after last team response:


\* [3785](https://github.com/microsoft/pylance-release/issues/3785 "auto-imports: `Self` type is imported from `typing` module, not `typing_extensions` in Python 3.9"): 3rd party responded 1 days ago but team last responded 332 days ago

---

## FOR ISSUES THAT ARE NOT MARKED AS BUGS:


### Issues in pylance-release that need a response from team:


\* [5448](https://github.com/microsoft/pylance-release/issues/5448 "Pylance Fail to read package source"): needs an initial team response (1 days old)

\* [5450](https://github.com/microsoft/pylance-release/issues/5450 "Python does not discover new value when subclassing flask config"): needs an initial team response (0 days old)

\* [5454](https://github.com/microsoft/pylance-release/issues/5454 "Cannot import sklearn.preprocessing.TargetEncoder"): needs an initial team response (0 days old)

### Issues in pylance-release that have comments from 3rd party after last team response:


\* [5377](https://github.com/microsoft/pylance-release/issues/5377 "Python and/or pylance is broken since last couple of updates "): 3rd party responded 0 days ago but team last responded 15 days ago
